Core.ApplicationServices/Organizations/Write/OrganizationWriteService.cs

```diff
@@ -70,12 +71,61 @@ private Result<Organization, OperationError> Update(Organization organization, O
         return result;
     }
 
+    public Result<Organization, OperationError> UpdateOrganization(Guid organizationUuid, OrganizationUpdateParameters parameters)
+    {
+        using var transaction = _transactionManager.Begin();
+        var result = _organizationService.GetOrganization(organizationUuid)
+            .Bind(organization => UpdateOrganizationIfWriteAccess(organization, parameters));
```

Comment:
did you manage to find a more "suitable name" for this like we discussed? Or is it part of a different pr

Reply:
That's in the other PR, I went with the split you suggested to avoid this branching name